cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,78 @@
-/// <reference types='cypress' />
+/* eslint-disable max-len */
+/* eslint-disable cypress/unsafe-to-chain-command */
+import { faker } from '@faker-js/faker';
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+  it('should handle pagination', () => {
+    cy.get('.-pageInfo').should('exist');
+    cy.get('.-previous > .-btn').should('exist');
+    cy.get('.-pageJump > input').should('exist');
+    cy.get('.-totalPages').should('exist');
+    cy.get('.-next > .-btn').should('exist');
+  });
+  it('should handle rows count selection', () => {
+    cy.get('select').should('exist');
+  });
+  it('should provide an ability to add a new worker.', () => {
+    const username = faker.internet.userName();
+    const email = faker.internet.email();
+    const lastname = faker.name.lastName();
+    cy.get('#addNewRecordButton').click();
 
+    cy.get('#firstName').click().type(username);
+    cy.get('#lastName').click().type(lastname);
+    cy.get('#userEmail').click().type(email);
+    cy.get('#age').click().type('23');
+    cy.get('#salary').click().type('500');
+    cy.get('#department').click().type('Legal');
+    cy.get('#submit').click();
+    cy.get(':nth-child(4) > .rt-tr > :nth-child(1)').should('exist');
+  });
+  it('should provide an ability to validate data in a worker row after creating a worker', () => {
+    const username = faker.internet.userName();
+    const email = faker.internet.email();
+    const lastname = faker.name.lastName();
+    cy.get('#addNewRecordButton').click();
+    cy.get('#firstName').click().type(username);
+    cy.get('#lastName').click().type(lastname);
+    cy.get('#userEmail').click().type(email);
+    cy.get('#age').click().type('23');
+    cy.get('#salary').click().type('500');
+    cy.get('#department').click().type('Legal');
+    cy.get('#submit').click();
+    cy.get(':nth-child(4) > .rt-tr > :nth-child(1)').should('contain', username, lastname, email);
```

Comment:
The assertion here is incorrect because `cy.get(':nth-child(4) > .rt-tr > :nth-child(1)')` is checking for a single element, and `should('contain', username, lastname, email)` is not a valid syntax for checking multiple values. You should check each value separately or use a different approach to validate the entire row content.